vmsdk/python/cctrusted_vm/sdk.py

```diff
@@ -157,6 +157,9 @@ def replay_cc_eventlog(self, event_logs:EventLogs) -> dict:
             Sample value:
                 { 0: { 12: <measurement_replayed>}}
         """
+        if event_logs is None:
+            LOG.error("Input event_logs should not be None.")
```

Comment:
IMO, it would be better to add check like this:
```
if event_logs is None or len(event_logs) == 0:
    LOG.info("No event log provided for replay.")
    return {}
```